community/modules/scheduler/schedmd-slurm-gcp-v6-controller/modules/slurm_files/variables.tf

```diff
@@ -264,6 +264,24 @@ variable "disable_default_mounts" {
   default     = false
 }
 
+variable "enable_controller_default_mounts" {
+  description = <<-EOD
+    Enable default global network storage to controller.
```

Comment:
this is also not very clear. From what I can tell, /apps is still shared through NFS from the controller, while /home is mounted from a GCS bucket. I am wondering what is the motivation for this, versus passing a "pre-existing-network-storage" to the controller...